website/js/app/app.js

```diff
@@ -0,0 +1,124 @@
+(function() {
+    // Prefix where HTML partials/templates will be found.
+    var prefix = '/google-cloud-powershell/js/app/controllers/';
+    
+    var app = angular.module('powershellSite', ['ngRoute', 'ngSanitize']);
+    app.config(function($routeProvider, $locationProvider) {
+        $routeProvider
+            .when('/',                           { controller: 'ContentController', templateUrl: prefix + 'content-homepage.ng' })
+            .when('/:product',                   { controller: 'ContentController', templateUrl: prefix + 'content-product.ng'  })
+            .when('/:product/:resource',         { controller: 'ContentController', templateUrl: prefix + 'content-resource.ng' })
+            .when('/:product/:resource/:cmdlet', { controller: 'ContentController', templateUrl: prefix + 'content-cmdlet.ng'   });
+        // Disabling because we don't control redirection, e.g. we cannot serve
+        // index.html for any URL under X. This means that  you'll get a 404
+        // for deep links. So, alas, we need the anchor.
+        $locationProvider.html5Mode(false);
+    });
+
+    // Directive for the left-nav, the table of contents.
+    app.directive('cmdletExplorer', function() {
+        return {
+            restrict: 'E',
+            templateUrl: prefix + 'cmdlet-explorer.ng',
+            controller: 'CmdletExplorerController',
+            controllerAs: 'cmdletExplorerCtrl',
+            scope: {
+                documentationObj: '='
```

Comment:
None of the bindings need to be two way. My understanding is that this syntax means "the directive has a variable on its scope called `documentationObj` and you set it by setting an attribute on the HTML element called `documentationObj`. Do I have that right?

I skimmed through the docs on directives and didn't see anything about data binding.
https://docs.angularjs.org/guide/directive

Is there another way I can express this to mean "make a one-way copy of the data"?